cql/src/main/java/org/apache/cassandra/stargate/metrics/ClientMetrics.java

```diff
@@ -54,6 +54,7 @@
 
   private AtomicInteger pausedConnections;
   private Meter requestDiscarded;
+  private Meter requestRate;
```

Comment:
nit: this `Meter` produces multiple outputs, so there's no single "rate" in the strict sense I guess. By analogy with `requestDiscarded`, would it be more readable to call the new `Meter` something like `requestsReceived`?

Reply:
Good suggestions. Thanks. wdyt about `requestsProcessed`? (it's in the current commit)

Reply:
sgtm :+1: